docs/bdd.md

```diff
@@ -281,7 +283,53 @@ Given('I have products in my cart', (table) => { // eslint-disable-line
   }
 });
 ```
+Examples of tables using: 
 
+```gherkin
+  Given I have a short employees card
+    | Harry | Potter  |
+    | Chuck | Norris  |
+```
+```js
+Given('I have a short employees card', (table) => {
+  const dataTableArgument = new CodeceptJS.DataTableArgument(table);
```

Comment:
please update the documentation and I'm merging :+1:

Reply:
It's in actual condition :)
’’’js
const dataTableArgument = new CodeceptJS.DataTableArgument(table);
’’’
For global ’DataTableArgument’ it gonna be like:

’’’js
const dataTableArgument = new DataTableArgument(table);
’’’
without any import above

Reply:
@EgorBodnar sorry, I had to be more explicit here.
I would prefer either global `DataTableArgument` or `const { DataTableArgument } = require('codeceptjs')` in docs

I mean, I prefer not to expose object `CodeceptJS` into userland nor advice it into documentation, because it was not mentioned anywhere else.

Reply:
Get it. I'll change the example in the doc

Reply:
@DavertMik
Updated the example in the doc in accordance with the requested edits.